sdcm/scylla_bench_thread.py

```diff
@@ -145,14 +146,31 @@ def verify_results(self):
 
         return sb_summary, errors
 
-    def create_stress_cmd(self, stress_cmd):
+    def create_stress_cmd(self, stress_cmd, cmd_runner):
         if self.connection_bundle_file:
             stress_cmd = f'{stress_cmd.strip()} -cloud-config-path={self.target_connection_bundle_file}'
         else:
             # Select first seed node to send the scylla-bench cmds
             ips = ",".join([n.cql_address for n in self.node_list])
             stress_cmd = f'{stress_cmd.strip()} -nodes {ips}'
 
+            if self.params.get("client_encrypt"):
+                loader = self.loader_set.get_loader()
```

Comment:
I rather not use `get_loader()` function, it's would interfere with the order of selections of loader nodes.

just pick the first one, if it's not important to have this specific one

Reply:
you can pass in the loader from where this function is called

Reply:
done